src/vulnpy/trigger/sqli.py

```diff
@@ -0,0 +1,60 @@
+import sqlite3
+
+db_connection = sqlite3.connect(":memory:")
+SELECT_ALL = "SELECT * FROM Character"
+
+
+def _db_reset():
+    db_connection.executescript(
+        """
+        DROP TABLE IF EXISTS Character;
+        CREATE TABLE Character(value, count);
+        INSERT INTO Character VALUES
+            ('a', '3'),('b', '5'),('c', '1')
+        """
+    )
+    db_connection.commit()
+
+
+def _execute(db_func):
+    try:
+        _db_reset()
+        cursor = db_connection.cursor()
+        db_func(cursor)
+        all_rows = cursor.execute(SELECT_ALL)
+        return ";".join([",".join(row) for row in all_rows])
+    except Exception:
+        return "<>"
```

Comment:
is the str value "<>" meaningful

Reply:
this is totally meaningless, but I didn't use `""` so that we could more easily distinguish this case from a successful query that just turned out to be empty. I can change it to `"error"` to make it more obvious if you like the sound of that